### ISSUE TITLE
Peer to peer code review

### ISSUE THREAD

**Issue by Lancelot-SO** [MAleemH/awesome-books#4]
> h1 tag should be used once in the project for page accessibility.

> Linters set properly

> Project meets the requirements
